src/components/Element/Article.jsx

```diff
@@ -0,0 +1,100 @@
+import axios from 'axios';
+import clsx from 'clsx';
+import { useState, useEffect, useRef } from 'react';
+import { Calendar, Tip, Content } from './ArticleElement';
+
+export function Article() {
+  const [article, setArticle] = useState([]);
+  const [sentence, setSentence] = useState({});
+  const [position, setPosition] = useState({});
+  const [link, setLink] = useState('');
+  const markRef = useRef(null);
+
+  useEffect(() => {
+    axios
+      .get(`${process.env.REACT_APP_URL}/api/article`)
+      .then((res) => {
+        setArticle(
+          res.data.articles.reduce((acc, curr) =>
+            curr.content.length > acc.content.length ? curr : acc
+          )
+        );
+      })
+      .catch((err) => {
```

Comment:
console.log is not suitable for production code. Please remove it. Also console logging error isn't a great way to handle error. It is okay for testing but usually if there's an error you'd want to handle it in someway such as trying again or informing the that service is currently not available.

Reply:
your mean like 
`throw new Error()`